### PR TITLE
Love 12 Support

### PR DESCRIPTION
**Pull request by JamesVanBoxtel**
Fix some image files that had invalid CRC metadata
Make the engine not crash when it can't load a file like that.

### CODE REVIEW

**Comment by Shosoul** [comment]
This may sound weird, but I feel like the engine _should_ crash if we try to use bad image files to let us know that the image is no good. It would probably crash from using a nil value anyways, but yea.

**Comment by JamesVanBoxtel** [comment]
Maybe? But we are going to have tons of complaints from old themes and we can't control those. Do we really want to keep fielding that question on discord?

File: graphics_util.lua [hunk 1]
```diff
@@ -1,7 +1,13 @@
 require("consts")
 
 local function load_img(path_and_name)
-  local img = love.image.newImageData(path_and_name)
+  print(path_and_name)
+  local img = nil
+  pcall(
```
Comment (Shosoul) [inline graphics_util.lua:6]:
```suggestion
  local status = pcall(
```

File: graphics_util.lua [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+    function()
+      img = love.image.newImageData(path_and_name)
+    end
+  )
```
Comment (Shosoul) [inline graphics_util.lua:10]:
```suggestion
  )
  if not status then
    print("Error loading " .. path_and_name)
  end
```

**Review by Shosoul — APPROVED** [review]

**Pull request merged**